src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/includeClassifier/index.js

```diff
@@ -16,7 +15,7 @@ const hasAmpVersion = href => {
   return !!hasAmpImageQueryString;
 };
 
-const includeClassifier = ({ href, pathname }) => {
+const includeClassifier = ({ href, isAmp }) => {
```

Comment:
```suggestion
const includeClassifier = ({ href, isAmpRequest }) => {
```
Just another rename suggestion, just makes clear this refers to the request we have received being for an amp page rather than the include being an 'amp include` if that makes sense?